### ISSUE TITLE
What are the next slides for v.2?

### ISSUE THREAD

**Issue by ttben** [PETILLON-Sebastien/facteurs_charge#38]
This issue is meant to talk globally about what is next in terms of slides.
So far, we have a map, a slide that shows the breakdown of the installations' production, and a slide that shows the breakdown of the installations' load.

**Comment by ttben** [comment]
- A slide that depicts the overall production, consumption, import, and export. All of this should be pretty much balanced. (Live example [here](http://jsfiddle.net/u3p5zyv8/))

- A slide that depicts the main source and the main destination of power for the current zone. Not sure how though. It can be as complex as [a connection map](https://www.d3-graph-gallery.com/graph/connectionmap_basic.html) though I wont opt for such complex visualization in the first place. A basic table of exchange will do IMO.

**Comment by PETILLON-Sebastien** [comment]
> A slide that depicts the overall production, consumption, import, and export.

Could it be a stacked area chart like [this](http://jsfiddle.net/peqkvzjg/) ?
Or it would be per hour in MWh or GWh (not an instantaneous power anymore).
I like this idea beacause it shows we need a constant balance.

> A slide that depicts the main source and the main destination of power for the current zone.

I think we need a visualisation of the exchanges, maybe with arrows between regions as electricitymap does ? But maybe in a rework.

**Comment by ttben** [comment]
> I like this idea beacause it shows we need a constant balance.

Yes. It is basically a continuous vision of the discrete barchart version :) But sure ;) 

--- 
> Or it would be per hour in MWh or GWh (not an instantaneous power anymore).

"Or"? 

----
>I think we need a visualisation of the exchanges, maybe with arrows between regions as electricitymap does ? But maybe in a rework.

Yes, as I answered
> I wont opt for such complex visualization in the first place. A basic table of exchange will do IMO.

Do you have an idea of how can we achieve such results? :) If so, I will be more incline to go for the shiny/risky option ;)

Another slide idea for the v.2 only?

**Comment by PETILLON-Sebastien** [comment]
> "Or"?

With the new chart it's ok. But if you're representing a production over a period, you have to use Wh units (with beautiful integrals of your curves).

> Do you have an idea of how can we achieve such results? :) If so, I will be more incline to go for the shiny/risky option ;)

Not yet, I should have a look at it.

> slide idea for the v.2 only?

Nope, I think we should keep iterative =D
I'll have to review the texts maybe, if you don't mind.

**Comment by ttben** [comment]
So is the discussion we are haviung on #39 part of v2?
Sorry to to have put this discussion on #39 and not here. I will delete my comments on #39 and add them below.

**Comment by ttben** [comment]
# Ranks

"This zone is the 1st in terms of consumption"
"This zone is the 2nd in terms of production"
"This zone is the 2nd in terms of SOLAR production"
"This zone is the 2nd in terms of exportation"

Proposition of logo for "production" (general term)
<img src="https://user-images.githubusercontent.com/6167712/83620917-4e8ca280-a58e-11ea-8bd2-e39b8032fbe6.jpg " width="70"/>


Proposition of logo for "consumption" (general term)
<img src="https://user-images.githubusercontent.com/6167712/83620994-66fcbd00-a58e-11ea-8497-c6d22aec0b93.jpg " width="70"/>

Export/import

|<img src="https://user-images.githubusercontent.com/6167712/83621136-94e20180-a58e-11ea-9a77-cde67348a828.png " width="70"/> |<img src="https://user-images.githubusercontent.com/6167712/83621153-9a3f4c00-a58e-11ea-805d-d5a561d8912a.png " width="70"/>|



# Criticality
Another idea, add an intuition of "criticality".
If a zone is the first producer, with a high load, and a number of dependent zones, it is most critical that the ones that only consumes, and produces nothing..

This is a load in term of "zone".
